.github/workflows/publish-release.yml

```diff
@@ -0,0 +1,101 @@
+name: Publish release
+
+on:
+  workflow_dispatch:
+
+jobs:
+  publish-release:
+    name: Publish Release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set version env variable
+        id: version-set
+        run: |
+          SHORT_VERSION=$((grep -w 'version' | cut -d= -f2) < gradle.properties
+          RELEASE_VERSION=$((grep -w 'shortVersion' | cut -d= -f2) < gradle.properties | rev | cut --complement -d- -f1 | rev)
```

Comment:
Shouldn't these two be swapped? i.e., `RELEASE_VERSION <= 'version'` and `SHORT_VERSION <= 'shortVersion'`?